.github/workflows/startup.yml

```
@@ -0,0 +1,35 @@
+
+jobs:
+    test-startup:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                node-version: [20, 21]
+                os: [ubuntu-latest, windows-latest, macos-latest]
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)